packages/openneuro-app/src/scripts/utils/__tests__/__snapshots__/markdown.spec.tsx.snap

```diff
@@ -0,0 +1,47 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Test <Markdown> component > allows a href with certain protocols 1`] = `
+<DocumentFragment>
+  <a
+    href="https://example.com"
+  >
+    Example link that should work.
+  </a>
+</DocumentFragment>
+`;
+
+exports[`Test <Markdown> component > does not allow href with unknown protocols 1`] = `
+<DocumentFragment>
+  <a>
+    Example link that should work.
+  </a>
+</DocumentFragment>
+`;
+
+exports[`Test <Markdown> component > filters out disallowed tags 1`] = `
+<DocumentFragment>
+  <ul>
+    <li>
+      Markdown document
+    </li>
+  </ul>
+</DocumentFragment>
+`;
+
+exports[`Test <Markdown> component > safely handles broken HTML tags 1`] = `
+<DocumentFragment>
+  <ul>
+    <li>
+      Markdown document
+
+      <br />
+      <br />
+      <ul>
+        <li>
+          test content
+        </li>
+      </ul>
+    </li>
+  </ul>
+</DocumentFragment>
+`;
```

Comment:
```suggestion
`;

exports[`Test <Markdown> component > filters close-break tags 1`] = `
<DocumentFragment>
  <br>
  sample text
</DocumentFragment>
`;
```

Reply:
```
exports[`Test <Markdown> component > filters close-break tags 1`] = `
<DocumentFragment>
  <span>
    <br />
    sample text
    <br />
  </span>
</DocumentFragment>
`;
```

This is the result for that input, kind of an interesting choice for fixing that syntax.

Reply:
It's probably closer to the original intent, so that's fair. I was just making a guess as to what it would look like.